webapp/src/test/java/org/opentestsystem/rdw/admin/CsvGenerator.java

```diff
@@ -0,0 +1,116 @@
+package org.opentestsystem.rdw.admin;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.google.common.collect.Lists.newArrayList;
+import static org.opentestsystem.rdw.admin.service.impl.DefaultCsvValidationService.HeaderGroup;
+import static org.opentestsystem.rdw.admin.service.impl.DefaultCsvValidationService.HeaderSSID;
+import static org.opentestsystem.rdw.admin.service.impl.DefaultCsvValidationService.HeaderSchool;
+import static org.opentestsystem.rdw.admin.service.impl.DefaultCsvValidationService.HeaderSubject;
+import static org.opentestsystem.rdw.admin.service.impl.DefaultCsvValidationService.HeaderUser;
+import static org.opentestsystem.rdw.admin.service.impl.DefaultCsvValidationService.HeaderYear;
+import static org.opentestsystem.rdw.admin.service.impl.DefaultCsvValidationService.RequiredHeaders;
+
+/**
+ * This testing utility class can be used to generate arbitrarily-large CSV sample files.
+ * Replace the school ids list with valid values from your local database.
+ */
+public class CsvGenerator {
```

Comment:
I decided to upload this little generator utility in case anyone else would like to generate test CSVs.